### ISSUE TITLE
Fix incorrect accessibility namespace

### ISSUE THREAD

**Issue by swudged** [yoyodesign/hako#15]
The accessibility helper and default setting currently looks for an incorrect namespace. This is a simply a typo and is missing one letter s on the word accessibility.
